### ISSUE TITLE
Adding Uninstall Commands for macOS and Linux.

### ISSUE THREAD

**Issue by NotMoni** [microsoft/vscode-docs#3454]
In, `docs/setup/setup-overview.md` the last line has methods to uninstall VSCode. Can we include a command for macOS and Linux eg`sudo rm -rf  PATH TO CODE`. I would be beneficial because we are talking about someone new to VSCode probability new to coding in general.

**Comment by profread** [comment]
guess the task is well spelled out is this assigned yet?

**Comment by profread** [comment]
@NotMoni instead of the one earlier stated ?

**Comment by NotMoni** [comment]
I'm sorry. I don't understand you. Can you rephrase your question please ?

**Comment by gregvanl** [comment]
Hi @NotMoni Thank you for opening an issue. We prefer to leave platform operations general (saying "Delete blah") rather than documenting a specific shell command or UI gesture.

**Comment by NotMoni** [comment]
Ok @gregvanl
